data/helpers.go

```diff
@@ -0,0 +1,16 @@
+package data
+
+import (
+	"time"
+
+	"go.viam.com/rdk/utils"
+)
+
+// GetDurationFromHz gets time.Duration from hz.
+func GetDurationFromHz(captureFrequencyHz float32) time.Duration {
+	// this would be once every ~11.5 days. This is close enough to 0
+	if utils.Float32AlmostEqual(captureFrequencyHz, 0, 0.000001) {
```

Comment:
can this epsilon value be a constant? maybe even a public you can reuse in schedulerEnabled

Reply:
I don't think we can unfortunately as `Float32AlmostEqual` takes a flaot32 and `Float64AlmostEqual` (which is used in schedulerEnabled) uses a float64 so it can't be the same variable

Reply:
ugh. ack